src/services/OfficeRnDDataAggregator.tsx

```diff
@@ -0,0 +1,49 @@
+import { OfficeRndBooking, AppBooking } from "./OfficeRnDTypes/Booking";
+import { OfficeRnDTeam } from "./OfficeRnDTypes/Team";
+import { OfficeRndMeetingRoom } from "./OfficeRnDTypes/MeetingRoom";
+import { OfficeRnDFloor } from "./OfficeRnDTypes/Floor";
+import { keyBy } from "../helpers/keyBy";
+
+export class OfficeRnDDataAggregator {
+  combineOfficeRnDData = (
```

Comment:
Good job, good PR
Just one remark about naming, aim to be simple and very explicit

To goal is for a dev looking at combineOfficeRnDData function to understand the end result from the name of the function only
Here for example "createAppBookings" makes more sense or something of the kind "createAppBookingsFromOfficeRnDData"

Reply:
Renamed! Merging